src/main/src/main/java/org/geoserver/catalog/LayerInfo.java

```diff
@@ -168,4 +168,28 @@
      */
     void setAdvertised(boolean advertised);
 
+    /**
+     * The default interpolation method.
+     *
+     * <p>
+     * If not specifed (i.e. {@code null}), the service default will be used.
+     * </p>
+     */
+    String getDefaultInterpolationMethod();
```

Comment:
Btw, noticed WMSInfo.WMSInterpolation is available only in gs-wms, a similar enum could be created inside LayerInfo (and/or be stand alone and used in both cases, but it would get trickier to ensure backwards compatibility)